frontend/lib/src/components/shared/Dropdown/Selectbox.tsx

```diff
@@ -266,6 +275,7 @@ const Selectbox: React.FC<Props> = ({
           // Nudge the dropdown menu by 1px so the focus state doesn't get cut off
           Popover: {
             props: {
+              ignoreBoundary: isInSidebar,
```

Comment:
Make it possible for pop-up to go beyond the sidebar boundary. (Same in other input widgets below)